### DISCUSSION TITLE
Plans for the Release of Version 0.4.8

### DISCUSSION THREAD

**Discussion by 0Werbe0** [cloudbase/powershell-yaml#154] [General]
Hi,
Is there an estimated release date for the stable 0.4.8 version? I see prerelease versions are available, but I’d prefer to use a stable version in production.

**Comment by gabriel-samfira** [comment]
Hi!

A new version has been released. Apologies for the delay!

https://www.powershellgallery.com/packages/powershell-yaml/0.4.8
